crates/tests-e2e/tests/e2e_force_close_settled_channel.rs

```diff
@@ -0,0 +1,47 @@
+#![allow(clippy::unwrap_used)]
+
+use native::api::ChannelState;
+use native::api::Direction;
+use tests_e2e::app::force_close_dlc_channel;
+use tests_e2e::app::get_dlc_channels;
+use tests_e2e::app::refresh_wallet_info;
+use tests_e2e::app::submit_order;
+use tests_e2e::setup;
+use tests_e2e::setup::dummy_order;
+use tests_e2e::wait_until;
+
+#[tokio::test(flavor = "multi_thread")]
+#[ignore = "need to be run with 'just e2e' command"]
+#[should_panic]
+async fn can_force_close_settled_channel() {
+    let setup = setup::TestSetup::new_with_open_position().await;
+
+    let closing_order = {
+        let mut order = dummy_order();
+        order.direction = Direction::Short;
+        order
+    };
+
+    submit_order(closing_order.clone());
+    wait_until!(setup.app.rx.position_close().is_some());
```

Comment:
Note, this is not really reliable to judge whether the channel did actually get settled successfully as this event is already thrown at the before the `SettleFinalize` message is sent back to the coordinator.

Reply:
So you would just remove it? I think I just followed the same approach that we use in the other force-close e2e test, and then extended it to prove something more meaningful.

Reply:
I wouldn't remove it, but probably fetch the dlc channels and check if the coordinator really thinks the channel is settled.

Reply:
Maybe I can do that another time. I think it's a bit more involved to add this to the e2e tests.